src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersControllerIT.java

```diff
@@ -208,6 +209,7 @@ private void verifyDbCpoValues(UUID id,
             assertEquals(CREATED_BY_IDAM_MOCK, savedEntity.get().getCreatedBy());
             assertNotNull(savedEntity.get().getCreatedTimestamp());
             assertTrue(beforeCreateTimestamp.isBefore(savedEntity.get().getCreatedTimestamp()));
+            assertFalse(savedEntity.get().isHistoryExists());
```

Comment:
Change to `.andExpect(jsonPath("$.history_exists", is(HISTORY_EXISTS_DEFAULT)))` as we know it should default for a Create.

Reply:
Sorry cut-n-paste error on my part:  try `assertEquals(HISTORY_EXISTS_DEFAULT, savedEntity.get().isHistoryExists());`  rather than assertFalse: to highlight that we expect it to default.